planemo/shed_lint.py

```diff
@@ -290,14 +290,14 @@ def lint_repository_dependencies(realized_repository, lint_ctx):
 def lint_shed_yaml(realized_repository, lint_ctx):
     path = realized_repository.real_path
     shed_yaml = os.path.join(path, ".shed.yml")
-    if not os.path.exists(shed_yaml):
-        lint_ctx.info("No .shed.yml file found, skipping.")
+    if not os.path.exists(shed_yaml) and realized_repository.repository_type != REPO_TYPE_UNRESTRICTED:
```

Comment:
Not sure I understand this change?

Reply:
Was an attempt to fix the testing error for https://github.com/galaxyproject/planemo/tree/master/tests/data/repos/suite_1 which fails because there is no `.shed.yml` .. which is OK in this case .. or isn't it?